### PR TITLE
Adds javadocs to tests

### PR DESCRIPTION
**Pull request by pekellogg**
Leverages @DisplayName for Test methods to create a descriptive text of what the method should do. Used this as opposed to javadocs to see message in testing context. Added per Lauren D. recco via Code Review - 2

### CODE REVIEW

File: p1-backend/src/test/java/com/example/p1_backend/services/QuestionServiceTest.java
Comment (sbowe1) [inline p1-backend/src/test/java/com/example/p1_backend/services/QuestionServiceTest.java:1]: @DisplayName("Update question status to incorrect") -> "Update content of question"

File: p1-backend/src/test/java/com/example/p1_backend/services/UserServiceTest.java
Comment (sbowe1) [inline p1-backend/src/test/java/com/example/p1_backend/services/UserServiceTest.java:1]: @DisplayName("Return updated user if exists")- the result is a new token

**Review by sbowe1 — CHANGES_REQUESTED** [review]

File: p1-backend/src/test/java/com/example/p1_backend/services/QuestionServiceTest.java
Comment (pekellogg) [reply p1-backend/src/test/java/com/example/p1_backend/services/QuestionServiceTest.java:1]: Made requested change

File: p1-backend/src/test/java/com/example/p1_backend/services/UserServiceTest.java
Comment (pekellogg) [reply p1-backend/src/test/java/com/example/p1_backend/services/UserServiceTest.java:1]: Looking into this now. Do you recall the method? Text, "Update question status to incorrect" is not found in UserServiceTest.java

Comment (pekellogg) [reply p1-backend/src/test/java/com/example/p1_backend/services/UserServiceTest.java:1]: UserServiceTest's update method: @DisplayName("Return updated user if exists") -> "Return updated token if user exists"

Comment (pekellogg) [reply p1-backend/src/test/java/com/example/p1_backend/services/UserServiceTest.java:1]: Resolved via Discord chatter

**Review by sbowe1 — APPROVED** [review]

**Pull request merged**